albumentations/augmentations/geometric/rotate.py

```diff
@@ -101,3 +101,68 @@ def apply_to_keypoint(self, keypoint, angle=0, **params):
 
     def get_transform_init_args_names(self):
         return ("limit", "interpolation", "border_mode", "value", "mask_value")
+
+
+class SafeRotate(DualTransform):
+    """Rotate the input inside the input's frame by an angle selected randomly from the uniform distribution.
```

Comment:
Please, also describe in the docs that result image may have some artifacts. After rotation, the image may have a different aspect ratio, and after resizing, it returns to its original shape with the original aspect ratio of the image. For these reason we may see some artifacts.